dev/README_RELEASE_AIRFLOW.md

```diff
@@ -315,11 +315,11 @@ The Release Candidate artifacts we vote upon should be the exact ones we vote ag
 
 - Create issue in github for testing the release using this subject:
 
-  ```shell script
-  cat <<EOF
-  Status of testing of Apache Airflow {VERSION}
-  EOF
-  ```
```

Comment:
nit: Any reason for this change?

This change before puts the code-block under the bullets with same indent

Reply:
It wasn't looking good on my IDE. The 'cat' command was not coloured. I will revert this change since it moved out of the indentation